app/views/static_pages/dashboard_steps/moderation_matrix.md

```diff
@@ -42,7 +50,7 @@ Medium severity violations such as:
   * Naming & shaming
 * Self promo without permission (ask with Modmail)
   * Do not share resources you have created solely for personal or monetary gain
-  * Examples: "Fill out my survey" (does not help the goals of Odin), "Join my boot camp/program/server" (Could mislead learners), "Why PHP is superior" (unrelated blog post)
+  * Examples: "Fill out my survey" (does not help the goals of Odin), "Join my boot camp/program/server" (Could mislead learners), "Why PHP is superior" (unrelated blog post), job recruitment
```

Comment:
I think we might want to move the `job recruitment` part out of examples and make it its own bullet point like the `Do not share resources (...)` one, like:

* Do not recruit for jobs.

Since it clearly is not an example like the previous ones.

Given that other server recruitment is given as an example here I would like to have it clearly outlined here as well - the current rules fail to clearly state that inviting people to own discord servers is forbidden and we currently have a total embargo on discord invite links. Something along the lines of such bullet point:

* Do not recruit for your own servers

Or alternatively, we could just combine these and have a bullet point that says something like:

* This includes recruitment of all kinds

Which seems to encompass `Join my boot camp/program/server/workplace` pretty well.

Reply:
Good points. Again, since this is changing up the matrix, I am not sure I want to do that here. Will think on this more.

Reply:
@scheals For both of these suggestions, I think they should first be addressed in the moderation-proposals channel before making the changes here. That way all moderators are away of the idea and can weigh in on them, as opposed to just the two of us making the change. Would you like to post them there, or do you want me to do it?

Reply:
@rlmoser99 Sounds good, I'll get to it then.